File: bookManageFrontDev/src/services/FirstItemSystemInfoController.js
import request from '@/utils/request' 

/*
*@param: file:interface org.springframework.web.multipart.MultipartFile@param: mid:int
*/
export function insertSystemInfoOfExcel(file,mid) {
    return request({
        url: request.baseUrl + `/insertSystemInfoOfExcel`,
        method: 'POST',
        data:file,
        params:{
            mid:mid,

        }
    })
}
/*
*@param: widList:java.util.List<java.lang.Integer>
*/
export function deleteSystemInfoOfList(widList) {
    return request({
        url: request.baseUrl + `/deleteSystemInfoOfList`,
        method: 'POST',
        data:widList
    })
}
/*
*@param: wid:class java.lang.Integer

*/
export function deleteByPrimaryKeySystemInfo(wid) {
    return request({
        url: request.baseUrl + `/deleteByPrimaryKeySystemInfo`,
        method: 'GET',
        params:{
            wid:wid,

        }
    })
}
/*
*@param: requestPageData:com.Httb.bookManage.util.RequestPageData<com.Httb.bookManage.mbg.entity.Firstitemsysteminfo>
*/
export function selectSystemInfoList(requestPageData) {
    return request({
        url: request.baseUrl + `/selectSystemInfoList`,
        method: 'POST',
        data:requestPageData
    })
}
/*
*@param: firstitemsysteminfo:class com.Httb.bookManage.mbg.entity.Firstitemsysteminfo
*/
export function insertSystemInfo(firstitemsysteminfo) {
    return request({
        url: request.baseUrl + `/insertSystemInfo`,
        method: 'POST',
        data:firstitemsysteminfo
    })
}

File: bookManageBackDev/src/main/java/com/Httb/bookManage/controller/FirstItemSystemInfoController.java
package com.Httb.bookManage.controller;

import com.Httb.bookManage.mbg.entity.Firstitemsysteminfo;
import com.Httb.bookManage.service.FirstItemSystemInfoService;
import com.Httb.bookManage.util.RequestPageData;
import com.Httb.bookManage.util.ResponsePageData;
import org.springframework.web.bind.annotation.*;
import org.springframework.web.multipart.MultipartFile;

import javax.annotation.Resource;
import java.util.List;

@RestController
public class FirstItemSystemInfoController {
    @Resource
    private FirstItemSystemInfoService firstItemSystemInfoService;

    @PostMapping("/selectSystemInfoList")
    public ResponsePageData<Firstitemsysteminfo> selectSystemInfoList(@RequestBody RequestPageData<Firstitemsysteminfo> requestPageData) {
        return firstItemSystemInfoService.selectSystemInfoList(requestPageData);
    }
    @PostMapping("/insertSystemInfo")
    public int insertSystemInfo(@RequestBody Firstitemsysteminfo firstitemsysteminfo) {
        return firstItemSystemInfoService.insert(firstitemsysteminfo);
    }
    @PostMapping("/insertSystemInfoOfExcel")
    public String insertSystemInfoOfExcel(@RequestParam int mid,@RequestBody MultipartFile file) throws Exception {
        return firstItemSystemInfoService.insertSystemInfoOfExcel(mid,file);
    }

    @GetMapping("/deleteByPrimaryKeySystemInfo")
    public int deleteByPrimaryKeySystemInfo(@RequestParam Integer wid) {
        return firstItemSystemInfoService.deleteByPrimaryKey(wid);
    }

    @PostMapping("/deleteSystemInfoOfList")
    public int deleteSystemInfoOfList(@RequestBody List<Integer> widList) {
        return firstItemSystemInfoService.deleteSystemInfoOfList(widList);
    }
}
